src/encoded/upgrade/donor.py

```diff
@@ -217,3 +217,26 @@ def fly_worm_donor_8_9(value, system):
     # https://encodedcc.atlassian.net/browse/ENCD-3616
     if value.get('parent_strains') == []:
         value.pop('parent_strains', None)
+
+
+@upgrade_step('fly_donor', '9', '10')
+def fly_donor_9_10(value, system):
+    # https://encodedcc.atlassian.net/browse/ENCD-4106
+    if 'aliases' in value:
+        for alias in value['aliases']:
+            if alias[0:6] == 'kyoto:':
+                new_alias = alias.replace(
+                    'kyoto:', 'dggr:')
```

Comment:
I will give `count=1`: https://docs.python.org/3.4/library/stdtypes.html#str.replace

I don't think there will be cases where "kyoto:" is in the "ID" but just to be safe and specific.

Same below.